kolibri_desktop_auth_plugin/kolibri_plugin.py

```diff
@@ -0,0 +1,12 @@
+from kolibri.plugins import KolibriPluginBase
+
+from . import settings
+
+
+class DaemonAuth(KolibriPluginBase):
+    translated_view_urls = "urls"
+    settings_module = settings
+
+    @property
+    def url_slug(self):
+        return r"daemon-auth/"
```

Comment:
The `deamon-auth/` URL prefix doesn't say much to me. I'd prefer if it includes something about "desktop user" or "system user". What about?: 
```suggestion
        return r"desktop-user-auth/"
```

Or maybe even better, leave it to the default which is a lower-case version of the plugin name. And rename the class name to DesktopAuth or DesktopUserAuth (I'm tempted to request a repo rename again to `kolibri-desktop-user-auth-plugin`).